gbpservice/nfp/orchestrator/modules/service_orchestrator.py

```diff
@@ -930,6 +972,14 @@ def handle_config_applied(self, event):
                  {'network_function_id':
                   network_function_id})
 
+    def handle_update_user_config_failed(self, event):
+        event_data = event.data
+        network_function_id = event_data['network_function_id']
+        LOG.error(_LE("NSO: updating user config failed, moving "
+                      "network function %(network_function_id)s to ERROR"),
+                  {'network_function_id': network_function_id})
+        self.handle_user_config_failed(event)
```

Comment:
If driver doesn't return stack_id, then NSO can't delete stack.
If stack_id is None, NSo won't trigger stack delete.